bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/handler/MagentaTVHandler.java

```diff
@@ -185,7 +189,7 @@ private void initializeThing() {
     }
 
     /**
-     * This routine is called every time the Thing configuration has been changed.
+     * This routine is called every time the Thing configuration has been changed (e.g. PaperUI)
```

Comment:
There is no such thing as a PaperUI anymore.
```suggestion
     * This routine is called every time the Thing configuration has been changed.
```

Reply:
reference removed